testutil/src/main/java/org/spine3/test/Verify.java

```diff
@@ -184,25 +184,37 @@ public static void fail(String message, Throwable cause) {
     }
 
     /**
-     * Asserts that two floats are not equal concerning a delta. If the expected value is infinity then the delta value
-     * is ignored.
+     * Asserts that two floats are not equal concerning a delta. If the expected value is infinity or NaN then
+     * the delta value is ignored.
      */
     public static void assertNotEquals(String itemName, float notExpected, float actual, float delta) {
         try {
             // handle infinity specially since subtracting to infinite values gives NaN and the
             // the following test fails
-            //noinspection FloatingPointEquality
-            if (Float.isInfinite(notExpected) && notExpected == actual || Math.abs(notExpected - actual) <= delta) {
+            if (areNaNs(notExpected, actual)
+                    || areSameKindOfInfinity(notExpected, actual)
+                    || Math.abs(notExpected - actual) <= delta) {
                 Assert.fail(itemName + SHOULD_NOT_BE_EQUAL + notExpected + '>');
             }
         } catch (AssertionError e) {
             throw mangledException(e);
         }
     }
 
+    private static boolean areNaNs(float first, float second)
+    {
```

Comment:
The bracket position is wrong (here and below). 

Please make sure you have the IDEA configured properly. And have a read on the codestyle we use. Spine wiki is the place to start searching.